### DISCUSSION TITLE
PhotoSync thumbnails

### DISCUSSION THREAD

**Discussion by 7eptem** [photoprism/photoprism#1853] [Q&A] [answered]
i was expected to get a thumbnail preview on PNG and JPG format but i dont, this make impossible to know what you want to import or to open, on JPG i get but not on all

**Comment by graciousgrey** [comment, marked as answer]
In which app do you miss thumbnails? In PhotoSync or PhotoPrism?
In case thumbnails are missing in PhotoPrism this checklist might help: https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Reply by 7eptem** [reply]
in PhotoSync

**Reply by graciousgrey** [reply]
PhotoSync is not developed by us.

**Reply by 7eptem** [reply]
> PhotoSync is not developed by us.

any alternatives for ios ?

**Reply by lastzero** [reply]
PhotoSync is available for iOS and Android: https://photoprism.app/partners

**Reply by 7eptem** [reply]
> PhotoSync is available for iOS and Android: https://photoprism.app/partners

thank you, but that was not i was looking for, i was looking for a alternative not a link to download (since this one is not displaying thumbnails properly

**Reply by lastzero** [reply]
👉 https://letmegooglethat.com/?q=ios+app+for+webdav+file+sync

**Reply by hholtmann** [reply]
The latest PhotoSync for iOS (4.6.5 and higher) version does add thumbnail support for JPEGs, HEIC and PNGs that do not contain an embedded preview. Thumbnails are automatically generated for file sizes up to 15MB by default. You can configure/customize this in the settings of the transfer target.
